src/cascadia/TerminalApp/VsDevCmdGenerator.h

```diff
@@ -0,0 +1,46 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- VsDevCmdGenerator
+
+Abstract:
+- Dynamic profile generator for Visual Studio Developer Command Prompt
+
+Author(s):
+- Charles Willis - October 2020
+
+--*/
+
+#pragma once
+#include "BaseVisualStudioGenerator.h"
+
+namespace TerminalApp
+{
+    class VsDevCmdGenerator :
+        public TerminalApp::BaseVisualStudioGenerator
+    {
+    public:
+        VsDevCmdGenerator() = default;
+        ~VsDevCmdGenerator() = default;
+
+        std::wstring_view GetNamespace() override
+        {
+            return std::wstring_view{ L"Windows.Terminal.VisualStudio.CommandPrompt" };
+        }
+
+        inline bool IsInstanceValid(const VsSetupConfiguration::VsSetupInstance instance) const override
+        {
+            return instance.VersionInRange(L"[15.0.0.0,)");

```

Comment:
This is unnecessary. The COM server only supports Visual Studio 15.0 and newer anyway. `-legacy` support in vswhere uses registry-based detection. Eliding the version range will improve the discovery performance by not having to parse or compare the versions.

Reply:
For context, I was the architect on the VS setup engine, wrote the setup configuration APIs, and wrote/still maintain vswhere.

Reply:
I didn't see a change here. Startup performance of Terminal is important and eliding both this and the comment below re: "16.2.0.0" to "16.2" will help.

Reply:
All the files moved unfortunately.